config-example/testvaults.txt

```diff
@@ -0,0 +1,7 @@
+# List of map elements to load for the map "test_vault.dmm", leave the file below this line blank to load all vaults.
+
+/datum/map_element/vault/clownroid
+/datum/map_element/vault/sokoban
+/datum/map_element/vault/black_site_prism
+/datum/map_element/dungeon/ninja_dojo
+/datum/map_element/dungeon/wizard_den
```

Comment:
Why not just make the default all vaults instead of this list?

Reply:
I feel like the most intuitive case is to have all vaults spawn yeah

Reply:
done that now